### PR TITLE
Gql02 user get by id ap itests kutovyi

### PR DESCRIPTION
**Pull request by rustamkutovyi**

### CODE REVIEW

File: src/modules/user/_tests/userGetById.spec.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+const {userCreateM, getUserByIdQ} = require ('./query')
+const {userCreateArg} = require('./data')
+const {gqlRequest} = require('../helper')
+const {expect} = require("chai");
+const generateId = require('../../../utils/generateId')
+const User = require ('../User')
+
+describe('GET USER BY ID', () => {
+    describe('GET USER BY ID - POSITIVE', () => {
+        let userId = null;
```
Comment (olgatbird) [inline src/modules/user/_tests/userGetById.spec.js:10]: Rustam, I would move this line "let userId = null" upper. Before describe with positive tests, so you would not need to type it again in describe with negative tests

File: src/modules/user/_tests/userGetById.spec.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            gqlRequest(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err)
+                    const respData = res.body
+                    expect(respData.data).to.eq(null)
+                    expect(respData.errors[0].message).to.eq('Cast to ObjectId failed for value "" (type string) at path "_id" for model "User"')
+                    done()
+                })
+        });
```
Comment (olgatbird) [inline src/modules/user/_tests/userGetById.spec.js:112]: Please, add schema related test, where you check how graphQl works with invalid queries and variables

**Review by olgatbird — CHANGES_REQUESTED** [review]

File: src/modules/user/_tests/userGetById.spec.js
[the remark this answers is not in the document]
Comment (rustamkutovyi) [reply src/modules/user/_tests/userGetById.spec.js:1]: Done

File: src/modules/user/_tests/userGetById.spec.js (hunk 1 above)
Comment (rustamkutovyi) [reply src/modules/user/_tests/userGetById.spec.js]: Done

File: src/modules/user/_tests/userGetById.spec.js (hunk 2 above)
Comment (rustamkutovyi) [reply src/modules/user/_tests/userGetById.spec.js]: Done

**Review by olgatbird — APPROVED** [review]

**Review by olgatbird — APPROVED** [review]

**Pull request merged**